AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyAssetEditor.py

```diff
@@ -53,9 +54,9 @@ def add_method_to_script_event(self, method_name: str) -> None:
         self.expand_category_by_name(DEFAULT_SCRIPT_EVENT)
         self.expand_category_by_name("Name")
 
-        self.update_new_method_name(method_name)
+        self.update_new_method_name(DEFAULT_METHOD_NAME, method_name)
 
-    def update_new_method_name(self, method_name: str) -> None:
+    def update_new_method_name(self, old_method_name: str, new_method_name: str) -> None:
```

Comment:
We should  update the name of this to something more meaningful like `update_script_event_method_name`

Reply:
fixed